### ISSUE TITLE
`OF column_name` syntax causing error in CREATE TRIGGER statement

### ISSUE THREAD

**Issue by adg-mh** [simolus3/moor#751]
Given the line below in a .moor file:

`CREATE TRIGGER trigger_name AFTER UPDATE OF some_column ON SomeTable`

the error below is caused:

```
line 7, column 45: Unknown column. These columns are available:
  ╷
7 │ CREATE TRIGGER trigger_name AFTER UPDATE OF some_column ON SomeTable
```

**Comment by simolus3** [comment]
Good catch, thanks! This is fixed on `develop`, I'll cherry-pick that to `master` and publish the fix.

**Comment by simolus3** [comment]
Fixed in `sqlparser` version `0.10.1`. You can run `pub upgrade` to get this version.
